packages/shared/lib/core/profile/utils/getStorageDirectoryOfProfiles.ts

```diff
@@ -6,3 +6,11 @@ export const getStorageDirectoryOfProfiles = async (): Promise<string> => {
     const appPath = await Platform.getUserDataPath()
     return `${appPath}/${PROFILE_STORAGE_DIRECTORY}`
 }
+
+export let storageDirectoryOfProfiles = ''
```

Comment:
why `getStorageDirectoryOfProfiles` cant be used? I see it used in multiple places in the code, why do we need to change the logic for this specific case?